tfe/resource_tfe_policy_set.go

```diff
@@ -119,6 +174,31 @@ func resourceTFEPolicySetRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("organization", policySet.Organization.Name)
 	}
 
+	// Set VCS policy set options.
+	d.Set("policies_path", policySet.PoliciesPath)
```

Comment:
We should probably move this one to line 172 as it's just a simple assignment like the others listed there.